lib/tasks/test_security_helper.rb

```diff
@@ -85,7 +85,17 @@ def self.yarn_audit(format: "human")
       cmd = AwesomeSpawn.build_command_line("yarn npm audit", params)
       puts "**   command: #{cmd}"
 
-      system(cmd, options)
+      system(cmd, options).tap do |success|
+        # If the run failed due to a configuration error, the error message will appear
+        # in the json output, but not in json format, so let's detect and display.
+        if !success && format == "json"
+          begin
+            JSON.parse(log_file)
+          rescue JSON::ParserError
+            $stderr.puts log_file.read
```

Comment:
Does it only write to stdout?  Is system sufficient or do we need stderr too?

Reply:
The yarn npm audit run doesn't write anything to stdout nor stderr, but only into the JSON file.  If the JSON file is not parsable, it's likely the error message.

Here I just wrote to stderr for display purposes.

Reply:
actually, now that I wrote that there's a minor bug here that I need to fix.  One sec.

Reply:
Oh right, you're not even capturing stdout.  My bad.

Reply:
Ok fixed - I forgot in the normal failure case that the log file will have 1 json per line, so we can't parse the whole file at once.  Now the code just attempts to parse the first line, and if that fails, then we know the file is raw text and we can present it.